scripts/e2e-test.sh

```
@@ -0,0 +1,137 @@
+    status)
+        echo "📊 E2E Services Status:"
+        echo ""
+        docker-compose -f docker-compose.e2e.yml ps
+        ;;
+
+    logs)
+        SERVICE=${2:-}
+        if [ -z "$SERVICE" ]; then
+            docker-compose -f docker-compose.e2e.yml logs --tail=50 --follow
+        else
+            docker-compose -f docker-compose.e2e.yml logs --tail=50 --follow $SERVICE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)